### PR TITLE
Added action button and claim function

### PR DESCRIPTION
**Pull request by frostbournesb**

### CODE REVIEW

**Comment by greatdolphinls** [comment]
Please reject removing `// eslint-disable-next-line react-hooks/exhaustive-deps`. 
It will make a lot of conflicts with other guys' work

File: .eslintrc.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ module.exports = {
     'react/no-unknown-property': 0,
     'react/prop-types': 0,
     'react/react-in-jsx-scope': 0,
+    'react-hooks/exhaustive-deps': 0,
```
Comment (greatdolphinls) [inline .eslintrc.js]: Please remove this line. it is necessary for development. We have to see what dependencies are missing.

File: contexts/compound-and-earn-context.js [hunk 2]
```diff
@@ -25,7 +28,6 @@ export function CompoundAndEarnProvider({ children }) {
     {
       getBalanceInfosByPool();
     }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [pools, gauges, account]);
```
Comment (ghost) [inline contexts/compound-and-earn-context.js]: can you put the query "data" var here to trigger the useEffect update too?

File: parts/Compound/CompoundListDetail/index.js [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
+import CompoundDialogs from '../CompoundDialogs';
 import ApyCalculation from './ApyCalculation';
 import SnobAbyCalculation from './SnobAbyCalculation';
 import Total from './Total';
-import CompoundDialogs from '../CompoundDialogs';
-import GradientButton from 'components/UI/Buttons/GradientButton';
-import { useCompoundAndEarnContract } from 'contexts/compound-and-earn-context';
 
 const useStyles = makeStyles((theme) => ({
   root: {
```
Comment (bigwampa) [inline parts/Compound/CompoundListDetail/index.js:15]: This doesn't seem to import the ABIs @frostbournesb

**Pull request closed without merging**